.github/workflows/publish-image.yaml

```diff
@@ -0,0 +1,29 @@
+name: Release Docker images
+on:
+  push:
+    branches:
+    - 'develop'
```

Comment:
Well
- our current 'pro' is running in nitric-dev project - it is "dev"
- if the cli image really was only produced via main, I think it would be too slow getting fixes into prod..

Reply:
@asalkeld we wouldn't just produce the image on main, we would always produce on devdelop/main, but the image tags would indicate the stability/version of the image. E.g. dev -> rc-latest (which we could still use with pro) and main -> latest.

This is a rough example as it duplicates the publish action (normally I would conditionally set the tags instead, but this this way is easier than figuring out the multi-line input for now and demonstrates how it's possible).
```yaml
- name: Push Nitric CLI Release Candidate Dockerhub
  if: github.ref == 'refs/heads/develop'
  uses: docker/build-push-action@v3
  with:
    context: .
    file: ./Dockerfile
    push: true
    tags: |
      nitrictech/cli:rc-latest
      nitrictech/cli:rc-x64
- name: Push Nitric CLI Latest Dockerhub
  if: github.ref == 'refs/heads/main'
  uses: docker/build-push-action@v3
  with:
    context: .
    file: ./Dockerfile
    push: true
    tags: |
      nitrictech/cli:latest
      nitrictech/cli:x64
```
While we don't advertise the existence of the CLI docker image yet it's still public, so if we can tag the images to best reflect the release of the build in the image I think we should.

Reply:
ok, thanks. I'll do that